user.rb

```diff
@@ -0,0 +1,17 @@
+class User
+  attr_accessor :name, :pin, :balance
+
+  def initialize(name, pin, balance)
+    @name    = name
+    @pin     = pin
+    @balance = balance.to_i
```

Comment:
if you have `attr_accessor` for each of these attributes, it is generally accepted that you should use the getter & setter methods associated with each, like this:
```ruby
self.name = name
self.pin = pin
self.balance = balance
```